deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql

```diff
@@ -103,35 +103,68 @@
         }],
 [ ]     default: []
       },
-[ ]   createdAt: {
-[ ]     type: Date,
-[ ]     default: Date.now
+[?]   createdAt: {
+[?]     type: Date,
+[?]     default: Date.now
       },
-[ ]   updatedAt: {
-[ ]     type: Date,
-[ ]     default: Date.now
+[?]   updatedAt: {
+[?]     type: Date,
+[?]     default: Date.now
       },
-[ ]   isEnabled: {
-[ ]     type: Boolean,
-[ ]     default: false,
+[X]   isEnabled: {
+[X]     type: Boolean,
+[X]     default: false,
 [-]     index: true
       },
-[ ]   reviewedBy: {
-[ ]     type: String,
-[ ]     default: null,
+[X]   reviewedBy: {
+[X]     type: String,
+[X]     default: null,
 [-]     index: true
       },
-[ ]   tags: {
-[ ]     type: Array,
+[X]   tags: {
+[X]     type: Array,
 [-]     index: true
       },
-[ ]   deleted: {
-[ ]     type: Boolean,
-[ ]     default: false,
+[X]   deleted: {
+[X]     type: Boolean,
+[X]     default: false,
 [-]     index: true
       },
-[ ]   wasSeeded: { type: Boolean }
+[-]   wasSeeded: { type: Boolean }
     }
+
+undocumented: followerIds
+db.getCollection('organizations').find({followerIds: {$exists: true, $not: {$size: 0}}}) = zero records
 */
 
-CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL_FILE}") YIELD value as organisation;
+CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL_FILE}") YIELD value as organization
+MERGE (o:Organization {id: organization._id["$oid"]})
+ON CREATE SET
+o.name                  = organization.name,
+o.slug                  = organization.slug,
+o.type                  = organization.type,
+o.description           = organization.description,
+o.descriptionExcerpt    = organization.descriptionExcerpt,
+o.language              = organization.language,
+o.avatar                = replace(organization.logo, 'https://api-alpha.human-connection.org', ''),
+o.coverImg              = replace(organization.coverImg, 'https://api-alpha.human-connection.org', ''),
+o.deleted               = COALESCE(organization.deleted,false),
+o.disabled              = NOT organization.isEnabled,
+o.createdAt             = datetime(organization.createdAt.`$date`),
+o.updatedAt             = datetime(organization.updatedAt.`$date`)
+WITH o, organization
+MATCH (u:User {id: organization.userId})
+MERGE (u)-[:CREATED]->(o)
+MERGE (u)-[:OWNS]->(o)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-13T23:21:55Z" title="Sunday, July 14th 2019, 1:21:55 am +02:00">Jul 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -103,35 +103,68 @@
         }],
 [ ]     default: []
       },
-[ ]   createdAt: {
-[ ]     type: Date,
-[ ]     default: Date.now
+[?]   createdAt: {
+[?]     type: Date,
+[?]     default: Date.now
       },
-[ ]   updatedAt: {
-[ ]     type: Date,
-[ ]     default: Date.now
+[?]   updatedAt: {
+[?]     type: Date,
+[?]     default: Date.now
       },
-[ ]   isEnabled: {
-[ ]     type: Boolean,
-[ ]     default: false,
+[X]   isEnabled: {
+[X]     type: Boolean,
+[X]     default: false,
 [-]     index: true
       },
-[ ]   reviewedBy: {
-[ ]     type: String,
-[ ]     default: null,
+[X]   reviewedBy: {
+[X]     type: String,
+[X]     default: null,
 [-]     index: true
       },
-[ ]   tags: {
-[ ]     type: Array,
+[X]   tags: {
+[X]     type: Array,
 [-]     index: true
       },
-[ ]   deleted: {
-[ ]     type: Boolean,
-[ ]     default: false,
+[X]   deleted: {
+[X]     type: Boolean,
+[X]     default: false,
 [-]     index: true
       },
-[ ]   wasSeeded: { type: Boolean }
+[-]   wasSeeded: { type: Boolean }
     }
+
+undocumented: followerIds
+db.getCollection('organizations').find({followerIds: {$exists: true, $not: {$size: 0}}}) = zero records
 */
 
-CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL_FILE}") YIELD value as organisation;
+CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL_FILE}") YIELD value as organization
+MERGE (o:Organization {id: organization._id["$oid"]})
+ON CREATE SET
+o.name                  = organization.name,
+o.slug                  = organization.slug,
+o.type                  = organization.type,
+o.description           = organization.description,
+o.descriptionExcerpt    = organization.descriptionExcerpt,
+o.language              = organization.language,
+o.avatar                = replace(organization.logo, 'https://api-alpha.human-connection.org', ''),
+o.coverImg              = replace(organization.coverImg, 'https://api-alpha.human-connection.org', ''),
+o.deleted               = COALESCE(organization.deleted,false),
+o.disabled              = NOT organization.isEnabled,
+o.createdAt             = datetime(organization.createdAt.`$date`),
+o.updatedAt             = datetime(organization.updatedAt.`$date`)
+WITH o, organization
+MATCH (u:User {id: organization.userId})
+MERGE (u)-[:CREATED_ORGA]->(o)
+MERGE (u)-[:OWNING_ORGA]->(o)
```


```suggestion
MERGE (u)-[:OWNS]->(o)
```
I don't like abbreviations. At first you think abbreviations are cool, but they always create confusion.